src/test/java/examples/menu/CustomMenuInputScannerTest.java

```diff
@@ -0,0 +1,23 @@
+package examples.menu;
+
+import org.academiadecodigo.bootcamp.Prompt;
+import org.academiadecodigo.bootcamp.scanners.menu.CustomMenuInputScanner;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class CustomMenuInputScannerTest {
+    public static void main(String[] args) {
+        Prompt prompt = new Prompt(System.in,System.out);
+
+        Map<Integer,String> options = new HashMap<>();
+        options.put(5,"Bahia");
+        options.put(30,"João");
+        options.put(2,"Tiago");
```

Comment:
Following the previous comment, if you wanted to assure that the menu would show the options with ascending order, you could use a `TreeMap` here. If you wanted to keep the order you're adding the options, you could use a `LinkedHashMap`. Either way would work with this implementation.